account_mtd/controllers/main.py

```diff
@@ -0,0 +1,33 @@
+import base64
+from odoo import http
+from datetime import datetime
+from werkzeug.utils import redirect
+
+class Authorize(http.Controller):
+    
+    @http.route('/auth-redirect', type='http', methods=['GET'])
+    def get_user_authorization(self, **args):
+        
+        #to determine which API the authorization code has been received for.
+        api_tracker = http.request.env['mtd.api_request_tracker'].search([('request_sent', '=', True),('request_received', '=', False)])
+        if len(api_tracker) > 1:
+            """We should never be in position in the first place as when the tokens are requested we are checking to see
+            whether there is already a request in process if so then we have to wait"""
+            #Work out which record we need to use make sure the record was created within last 10 mins
+            now_datetime = datetime.now()
+            time_10_mins_ago = now_datetime.timedelta(minutes=10)
```

Comment:
as `now_datetime` is only used once:
`time_10_mins_ago = datetime.now().timedelta(minutes=10)`

However, it looks as if time_10_mins_ago isn't used.  A line should probably be added (just for readability on missing features):
`# TODO: Implement <x> feature` <-- Based on whatever feature is supposed to go here